centreon/www/front_src/src/Configuration/ConfigurationBase/Dialogs/Delete/Delete.tsx

```diff
@@ -0,0 +1,40 @@
+import { Typography } from '@mui/material';
+
+import { useTranslation } from 'react-i18next';
+
+import { Modal } from '@centreon/ui/components';
+
+import { labelCancel, labelDelete } from '../../translatedLabels';
+
+import { sanitizedHTML } from '@centreon/ui';
+import useDelete from './useDelete';
+
+const DeleteDialog = (): JSX.Element => {
+  const { t } = useTranslation();
+
+  const { close, confirm, isMutating, isOpened, headerContent, bodyContent } =
+    useDelete();
+
+  return (
+    <Modal open={isOpened} size="large" onClose={close}>
+      <Modal.Header>{headerContent}</Modal.Header>
+      <Modal.Body>
+        <Typography>
+          {sanitizedHTML({ initialContent: bodyContent })}
```

Comment:
Use the Trans component instead: https://react.i18next.com/latest/trans-component
Something like this
```
<Trans
  i18nKey="myKey" // optional -> fallbacks to defaults if not provided
  defaults="hello <italic>beautiful</italic> <bold>{{what}}</bold>" // optional defaultValue
  values={{ what: 'world'}}
  components={{ italic: <i />, bold: <strong /> }}
/>

Reply:
same as above